src/sentry/integrations/message_builder.py

```diff
@@ -65,7 +65,10 @@ def build_attachment_title(obj: Group | GroupEvent) -> str:
     title = obj.title
 
     if ev_type == "error" and "type" in ev_metadata:
-        title = ev_metadata["type"]
+        if "title" in ev_metadata:
+            title = ev_metadata["title"]
```

Comment:
I don't know if this is a terrible idea like if other SDKs frequently provide `title` in the event metadata and this would have an adverse effect.